common/src/test/java/org/dash/wallet/common/TransactionFilterTests.kt

```diff
@@ -22,7 +22,7 @@ import org.bitcoinj.core.*
 import org.bitcoinj.params.TestNet3Params
 import org.dash.wallet.common.transactions.CoinsFromAddressTxFilter
 import org.dash.wallet.common.transactions.CoinsToAddressTxFilter
-import org.dashj.bls.Utils
+import org.bitcoinj.core.Utils
```

Comment:
This test uses `Utils.HEX.encode()` which is in `dashj` and `dashj-bls`, but with `dashj 0.18`, `dashj-bls` is no longer bundled with `dashj`.